taxonomies/categories.txt

```diff
@@ -22588,31 +22587,68 @@ fr:Crèmes fraîches légères, Crème fraîche légère, crème légère
 <en:Sour creams
 fr:Crèmes fraîches liquides, Crème fraîche liquide, Crème liquide
 
+# Consider making this a synonym for Sour creams.
 <en:Sour creams
-en:Smetana
-de:Schmand
 fr:Smetana
 nl:Smetana
-ru:Сметана
-wikidata:ru:Q326061
+wikidata:en:Q326061
```

Comment:
I rather keep smetana as a separate category, but as a child of "en:fermented creams".